### PR TITLE
Fix TestTimexDateHelpers unit tests

### PR DESCRIPTION
**Pull request by Batta32**

### CODE REVIEW

File: Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java [hunk 1]
```diff
@@ -249,80 +249,78 @@ private static String nextDateValue(TimexProperty timex, Calendar date) {
...
-        Integer daysOffset = DayOfWeek.MONDAY.getValue() - jan1.get(Calendar.DAY_OF_WEEK);
-        Calendar firstWeekDay = jan1;
-        firstWeekDay.add(Calendar.DATE, daysOffset);
+        LocalDateTime jan1 = LocalDateTime.of(year, 1, 1, 0, 0);
+        Integer daysOffset = DayOfWeek.MONDAY.getValue() - jan1.getDayOfWeek().getValue();
+        LocalDateTime firstWeekDay = jan1;
+        firstWeekDay = firstWeekDay.plusDays(daysOffset);
 
-        Integer firstWeek = jan1.get(Calendar.WEEK_OF_YEAR);
+        Integer firstWeek = jan1.getDayOfYear();
```
Comment (fran893) [inline Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java]: Is this correct? the removed line was getting the week and in the line you added you are getting the day of the year

**Review by fran893 — CHANGES_REQUESTED** [review]

**Review by VictorGrycuk — APPROVED** [review]
LGTM

File: Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java (hunk 1 above)
Comment (Batta32) [reply Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java]: You are right!

Comment (Batta32) [reply Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexResolver.java]: Applied!

**Review by fran893 — APPROVED** [review]

**Pull request merged**